lib/rdoc/generator/template/darkfish/_sidebar_toggle.rhtml

```diff
@@ -0,0 +1,26 @@
+<button id="navigation-toggle" class="navigation-toggle" aria-label="Toggle sidebar" aria-expanded="true" aria-controls="navigation">
+  <!--
+    Copyright (c) 2021 Luca Burgio
```

Comment:
I'm not sure this is the right way to attach the license.

Reply:
Alternatively, maybe we could use U+2630 (☰) as an icon.

Reply:
Or U+2261(≡)?

Reply:
Let's give `U+2630` a try? I'd prefer not importing additional asset for this.

Reply:
Updated [here](https://github.com/ruby/rdoc/pull/1025/commits/98f670e0fe55efb42554610cb195551cdd02cbec)